### PR TITLE
Use non mirror apache url

### PR DESCRIPTION
**Pull request by edgardmessias**
See tomcat for example

### CODE REVIEW

**Comment by edgardmessias** [comment]
Ping @Ash258

**Comment by r15ch13** [comment]
This will cause issues because PowerShell doesn't like redirects from HTTPS to HTTP.

**Comment by edgardmessias** [comment]
@r15ch13 , maybe change all URLs to use `https://www.apache.org/dist/`?

**Comment by edgardmessias** [comment]
@r15ch13 @Ash258, solved problem with redirect from HTTPS to HTTP, now, only redirect from HTTP to HTTP or HTTP to HTTPS

**Comment by Ash258** [comment]
I am boucing off this. Someone else review it.

- I don't use any of this
- I don't like http urls, when https are available, so this PR is totally unmergable for me.

**Comment by r15ch13** [comment]
I agree on this @Ash258, downgrading to http just because some mirrors don't support https or a dynamic mirror redirection that doesn't favour https shouldn't be an option.
It would be better to use a fixed mirror with https or try to catch the error on powershell 6 and tell the user that there was an attempt to redirect to http.

_Sent from my OnePlus5T using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by edgardmessias** [comment]
Changed to use default apache url

File: bucket/zeppelin.json [hunk 1]
```diff
@@ -17,10 +17,11 @@
         "re": "zeppelin-([\\d.]+)-bin-all.tgz"
     },
     "autoupdate": {
-        "url": "https://archive.apache.org/dist/zeppelin/zeppelin-$version/zeppelin-$version-bin-all.tgz#/zeppelin.tar.gz",
+        "url": "https://www.apache.org/dist/zeppelin/zeppelin-$version/zeppelin-$version-bin-all.tgz#/zeppelin.tar.gz",
         "extract_dir": "zeppelin-$version-bin-all",
         "hash": {
-            "url": "$url.sha512"
+            "url": "$url.sha512",
+            "find": "^($sha512)"
```
Comment (r15ch13) [inline bucket/zeppelin.json]: `"find": "^($sha512)"` is not required, scoop knows the file format and extracts the checksum automatically.

File: bucket/netbeans.json [hunk 2]
```diff
@@ -28,9 +28,10 @@
         "re": "Apache NetBeans ([\\d.]+)</a></li>"
     },
     "autoupdate": {
-        "url": "https://www-eu.apache.org/dist/incubator/netbeans/incubating-netbeans/incubating-$version/incubating-netbeans-$version-bin.zip",
+        "url": "https://www.apache.org/dist/incubator/netbeans/incubating-netbeans/incubating-$version/incubating-netbeans-$version-bin.zip",
         "hash": {
-            "url": "$url.sha512"
+            "url": "$url.sha512",
+            "find": "^($sha512)"
```
Comment (r15ch13) [inline bucket/netbeans.json]: `"find": "^($sha512)"` is not required, scoop knows the file format and extracts the checksum automatically.

**Pull request merged**